hieradata/site/dev.yaml

```diff
@@ -105,3 +105,7 @@ accounts::user_list:
 firewall::ensure: "stopped"
 
 letsencrypt::server: "https://acme-staging.api.letsencrypt.org/directory"  # testing url
+profile::core::icinga_agent::install: true
+profile::core::icinga_agent::icinga_master_fqdn: "icinga-master.ls.lsst.org"
+profile::core::icinga_agent::icinga_master_ip: "139.229.135.31"
+profile::core::icinga_resources::dhcp_server: "139.229.135.5"
```

Comment:
Isn't this the wrong dhcp server for dev?

Reply:
fixed